decentraland/unity-renderer
# feat: biw retrocompatibility assets

## What does this PR change?

This PR fixes #861. 

This PR enables retro compatibility with the old assets of the scene. This way, even if the assets are deleted from the catalog, if they still live in the content server, we will be able to use them 

## How to test the changes?


1. Play in the unity editor with this custom URL in the editor: https://play.decentraland.zone/branch/feat/biw-gltf-asset/index.html
2. Publish an old builder scene  that contains the old assets that don't exist anymore in the catalog
3. Enter in builder in world in the land that you have published
4. Everything should load as normal


We should be able to test if this work in this link but for some reason, it doesn't work
 https://play.decentraland.zone/branch/feat/biw-gltf-asset/index.html?ENABLE_BUILDER_IN_WORLD&renderer=urn:decentraland:off-chain:renderer-artifacts:feat/biw-assets&ENV=org

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
